source/sync/learn/overview.txt

```diff
@@ -160,8 +160,8 @@ Before You Start
 - If you don't already have one, `sign up for a free MongoDB Atlas account
   <https://www.mongodb.com/cloud/atlas/register?tck=docs_realm>`__.
 
-- Ensure that you have a MongoDB Atlas cluster that runs MongoDB version 4.4 or
-  later. You can use a free shared M0 cluster to explore and develop your app.
+- For Partition-Based Sync, ensure that you have a MongoDB Atlas cluster that runs MongoDB version 4.4 or
```

Comment:
This has now become the first mention of Partition-Based Sync and Flexible Sync on the page. Since we introduce them by name here, I'd recommend linking out to them directly so the reader can learn more - you can see the refs in lines 260 and 270 below - or adding an anchor above "Choose a Sync Mode" in line 258, and adding a link to that from here, ala:

> For more information, see: :ref:'Choose a Sync Mode <link-to-your-new-anchor>'.

Reply:
Thanks, @dacharyc! I've moved the first-time reference links up to the new references. I removed the old links to try to keep readers in the flow in those areas. I'm not very familiar with this page though, so let me know if keeping the old links makes more sense.